authority/config.go

```diff
@@ -91,6 +93,17 @@ func (c *AuthConfig) Validate(audiences provisioner.Audiences) error {
 	if c.Template == nil {
 		c.Template = &x509util.ASN1DN{}
 	}
+
+	if c.Backdate != nil {
+		if c.Backdate.Duration < 0 {
```

Comment:
I don't see any use case of having backdates in the future, you can already get a certificate in the future if you want to. And I prefer to enforce a format rather than using the absolute value.

Reply:
fair point